config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -149,7 +149,7 @@ periodics:
       - --gcp-zone=us-west1-b
       - --ginkgo-parallel=30
       - --provider=gce
-      - --test_args=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8
+      - --test_args=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[NodeFeature:RuntimeHandler\] --minStartupPods=8
```

Comment:
can we add a comment here or in the commit message about *why* we're skipping the runtimehandler.

Reply:
The difference is between test-infra/jobs/e2e_node/containerd/containerd-release-1.2/env
and test-infra/jobs/e2e_node/containerd/containerd-release-1.3/env

Containerd-1.2 does not support RuntimeHandler per:
CONTAINERD_EXTRA_RUNTIME_HANDLER: 'test-handler'
CONTAINERD_EXTRA_RUNTIME_OPTIONS: |
  BinaryName = "/home/containerd/usr/local/sbin/runc"

I am thinking where is the good place to add this comment.

Reply:
Also, by searching, I saw a lot of test suites has |\[NodeFeature:RuntimeHandler\]  without explaining. The best place explaining maybe in the file  test-infra/jobs/e2e_node/containerd/containerd-release-1.2/env file?